lib/datasource/remote/api/proposals_repository.dart

```diff
@@ -253,8 +254,9 @@ class ProposalsRepository extends NetworkRepository with EosRepository {
     return http
```

Comment:
The method should now be called "getDelegators" (plural) since it now returns a list.

Reply:
Renamed 👍